src/common/locations.ts

```diff
@@ -100,6 +111,13 @@ export function findCountyByFips(fips: string) {
   return undefined;
 }
 
+export function isValidState(fips: string): boolean {
```

Comment:
Feel like this should have "fips" in the name...  I was gonna' suggest `isStateFips`, but it looks like we already have that. 😁  So maybe either change `isStateFips` to include this functionality (making sure it's valid) or else this can be `isValidStateFips`.